src/BtmsStub/WireMockExtensions.cs

```diff
@@ -14,14 +14,21 @@ public static void StubSingleImportNotification(
         this WireMockServer wireMock,
         bool shouldFail = false,
         string chedReferenceNumber = ChedReferenceNumbers.ChedA,
-        Func<IRequestBuilder, IRequestBuilder>? transformRequest = null
+        Func<IRequestBuilder, IRequestBuilder>? transformRequest = null,
+        Func<string, string>? transformResponse = null
```

Comment:
Perhaps consider using `Func<JsonNode, JsonNode>?` instead so the parsing and returning as a string doesn't need to be handled by the consuming code.